schemas/documents/registry/creditClass.js

```diff
@@ -110,5 +110,15 @@ export default {
       type: 'reference',
       to: [{ type: 'program' }],
     },
+    {
+      title: 'Retirement Label',
```

Comment:
maybe worth adding the following to make it clearer for editor what it is:
```suggestion
      title: 'Retirement Certificate Label',
```